lib/BackupPC/CGI/Metrics.pm

```diff
@@ -309,7 +309,13 @@ sub action
                     }
 
                 } else {
-                    $content .= "$promKey $metrics{$section}{$entry}\n";
+                    my $val;
+                    if ( ref($metrics{hosts}{$host}{$entry}) eq "ARRAY" ) {
+                        $val = @{$metrics{hosts}{$host}{$entry}};
```

Comment:
$host is undefined when this line is executed.  What was intended by this change?

Perhaps @jooola can figure out what was intended.